src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -330,13 +352,21 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
                 {amountValue && <StyledText>{formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
                 {yieldPercent && (
                   <StyledText>
-                    {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
+                    {t('components.transaction.token-input.yield')}
+                    <ContrastText> {yieldPercent} </ContrastText>
                   </StyledText>
                 )}
               </TokenExtras>
             </TokenData>
           )}
         </TokenInfo>
+        {listItems?.length > 1 && displayGuidance && (
+          <ZapMessageContainer onClick={openSearchList}>
+            ⚡ {t('components.transaction.zap-guidance.part-1')}{' '}
```

Comment:
As stated on the json file comment, this should be a composed key/markdown to be correctly translated

Reply:
We can add this after merging